src/cli/Import.cpp

```diff
@@ -40,12 +41,20 @@
  *
  * @return EXIT_SUCCESS on success, or EXIT_FAILURE on failure
  */
+
+const QCommandLineOption Import::SetKeyFileOption =
```

Comment:
@Colfenor I think we should reuse the option as defined in `Create`, since they are exactly the same. Also in the future I believe we should centralize the database creation options and behavior, so that would be a step in that direction.

Reply:
jup, I changed it to using the `Create`. Dodging some code duplication :)